keps/sig-storage/625-csi-migration/README.md

```diff
@@ -229,7 +254,7 @@ Specifically, for each in-tree plugin corresponding CSI drivers, it will have
 ### Rollout, Upgrade and Rollback Planning
 
 * **How can a rollout fail? Can it impact already running workloads?**  
```

Comment:
This question should cover failures relating to `InTreePlugin{vendor}Unregister`. What if it's enabled when it shouldn't be? What about enabling or disabling at runtime?

Reply:
Done.  Thanks for the advice! I added some more description in the section you mentioned. Please let me know if that is sufficient.